plugin/Field/FieldModuleCollectionDecoratorInternalAnnotations.php

```diff
@@ -64,7 +64,7 @@ public function getFieldByModuleAndName(string $module, string $name): Field
 		$pField = parent::getFieldByModuleAndName($module, $name);
 		$newLabel = $this->getNewFields()[$module][$name] ?? null;
 		if ($newLabel !== null) {
-			$pField->setLabel(__($newLabel, 'onoffice-for-wp-websites'));
+			$pField->setLabel( sprintf( __( '%s', 'onoffice-for-wp-websites' ), $newLabel ) );
```

Comment:
How is this supposed to work? How should the WordPress translation know what `$newLabel` is to enable translation? :)

Reply:
it has been translated, for example line 49 just $newLabel is enough, it needs to be translated before using it.
Can you please test it? I tested it and it still translates normally :)
As for the words that have not been translated before, I will check them carefully in this PR

Reply:
If `$newLabel` is already translated, why are we using `__( '%s', 'onoffice-for-wp-websites' )` here? :)